pkg/cmd/datastore/datastore.go

```diff
@@ -213,6 +214,7 @@ func RegisterDatastoreFlagsWithPrefix(flagSet *pflag.FlagSet, prefix string, opt
 	flagSet.StringVar(&opts.SpannerEmulatorHost, flagName("datastore-spanner-emulator-host"), "", "URI of spanner emulator instance used for development and testing (e.g. localhost:9010)")
 	flagSet.Uint64Var(&opts.SpannerMinSessions, flagName("datastore-spanner-min-sessions"), 100, "minimum number of sessions across all Spanner gRPC connections the client can have at a given time")
 	flagSet.Uint64Var(&opts.SpannerMaxSessions, flagName("datastore-spanner-max-sessions"), 400, "maximum number of sessions across all Spanner gRPC connections the client can have at a given time")
+	flagSet.Uint64Var(&opts.SpannerEstimatedBytesPerRelationship, flagName("datastore-spanner-estimated-bytes-per-relationship"), spanner.DefaultEstimatedBytesPerRelationship, "estimated number of bytes per relationship tuple in the spanner instance")
```

Comment:
This does seem like a strange flag to have the user provide. While I understand the motivation of performance and simplifying the code, it feels clunky because it requires internal knowledge of how Spanner stores data on disk.

Even just querying Spanner for the size of a random stored relationship tuple when `Statistics()` is called would be probably be good enough to estimate the number of relationships, instead of providing this flag.

Reply:
It is odd, true. I added it as a flag so users can set it IF they want more accurate estimates, not that the estimates are used for much as-is :)

Reply:
Given how many datastore-specific flags we already have, avoiding adding more when it isn't absolutely necessary will reduce potential tech debt if we decide to consolidate the flags in the future.

Reply:
Think I should just hard code in then? Alternatively, we could make it load a random relationship and estimate based on that?

Reply:
Changed to load some relationships and calculate the estimated size based on them, with a fallback if none found